### PR TITLE
Adding spacing css utils files

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

Adding spacing css utils files
#
### Type of change :

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Review by chirag-madlani — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+.m-0 {
+  margin: 0 !important;
+}
+.m-xs {
+  margin: @margin-xs;
+}
+.m-sm {
+  margin: @margin-sm;
+}
+.m-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:23]: This should be `.m-md`

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+}
+.m-x-xs {
+  margin-right: @margin-xs;
+  margin-left: @margin-xs;
+}
+.m-x-sm {
+  margin-right: @margin-sm;
+  margin-left: @margin-sm;
+}
+.m-x-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:38]:
```suggestion
.m-x-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+}
+.m-y-xs {
+  margin-top: @margin-xs;
+  margin-bottom: @margin-xs;
+}
+.m-y-sm {
+  margin-top: @margin-sm;
+  margin-bottom: @margin-sm;
+}
+.m-y-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:54]:
```suggestion
.m-y-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+.m-r-0 {
+  margin-right: 0 !important;
+}
+.m-r-xs {
+  margin-right: @margin-xs;
+}
+.m-r-sm {
+  margin-right: @margin-sm;
+}
+.m-r-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:71]:
```suggestion
.m-r-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+.m-l-0 {
+  margin-left: 0 !important;
+}
+.m-l-xs {
+  margin-left: @margin-xs;
+}
+.m-l-sm {
+  margin-left: @margin-sm;
+}
+.m-l-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:86]:
```suggestion
.m-l-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+.m-t-0 {
+  margin-top: 0 !important;
+}
+.m-t-xs {
+  margin-top: @margin-xs;
+}
+.m-t-sm {
+  margin-top: @margin-sm;
+}
+.m-t-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:101]:
```suggestion
.m-t-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+.m-b-0 {
+  margin-bottom: 0 !important;
+}
+.m-b-xs {
+  margin-bottom: @margin-xs;
+}
+.m-b-sm {
+  margin-bottom: @margin-sm;
+}
+.m-b-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:116]:
```suggestion
.m-b-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 8]
```diff
@@ -0,0 +1,251 @@
...
+.p-0 {
+  padding: 0 !important;
+}
+.p-xs {
+  padding: @padding-xs !important;
+}
+.p-sm {
+  padding: @padding-sm;
+}
+.p-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:146]:
```suggestion
.p-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 9]
```diff
@@ -0,0 +1,251 @@
...
+}
+.p-x-xs {
+  padding-right: @padding-xs;
+  padding-left: @padding-xs;
+}
+.p-x-sm {
+  padding-right: @padding-sm;
+  padding-left: @padding-sm;
+}
+.p-x-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:160]:
```suggestion
.p-x-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 10]
```diff
@@ -0,0 +1,251 @@
...
+}
+.p-y-xs {
+  padding-top: @padding-xs;
+  padding-bottom: @padding-xs;
+}
+.p-y-sm {
+  padding-top: @padding-sm;
+  padding-bottom: @padding-sm;
+}
+.p-y-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:176]:
```suggestion
.p-y-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 11]
```diff
@@ -0,0 +1,251 @@
...
+.p-r-0 {
+  padding-right: 0 !important;
+}
+.p-r-xs {
+  padding-right: @padding-xs;
+}
+.p-r-sm {
+  padding-right: @padding-sm;
+}
+.p-r-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:197]:
```suggestion
.p-r-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 12]
```diff
@@ -0,0 +1,251 @@
...
+.p-l-0 {
+  padding-left: 0 !important;
+}
+.p-l-xs {
+  padding-left: @padding-xs;
+}
+.p-l-sm {
+  padding-left: @padding-sm;
+}
+.p-l-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:212]:
```suggestion
.p-l-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 13]
```diff
@@ -0,0 +1,251 @@
...
+.p-t-0 {
+  padding-top: 0 !important;
+}
+.p-t-xs {
+  padding-top: @padding-xs;
+}
+.p-t-sm {
+  padding-top: @padding-sm;
+}
+.p-t-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:227]:
```suggestion
.p-t-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 14]
```diff
@@ -0,0 +1,251 @@
...
+.p-b-0 {
+  padding-bottom: 0 !important;
+}
+.p-b-xs {
+  padding-bottom: @padding-xs;
+}
+.p-b-sm {
+  padding-bottom: @padding-sm;
+}
+.p-b-m {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:242]:
```suggestion
.p-b-md {
```

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 15]
```diff
@@ -0,0 +1,251 @@
...
+ */
+
+.m-0 {
+  margin: 0 !important;
+}
+.m-xs {
+  margin: @margin-xs;
+}
+.m-sm {
+  margin: @margin-sm;
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:21]: will this variable work? without importing?

**Comment by Ashish8689** [comment]
@Sachin-chaurasiya @ShaileshParmar11  All comment's has been addressed.

**Pull request merged**